src/cityreader/cityreader.py

```diff
@@ -59,13 +77,35 @@ def cityreader(cities=[]):
 # Tucson: (32.1558,-110.8777)
 # Salt Lake City: (40.7774,-111.9301)
 
-# TODO Get latitude and longitude values from the user
 
 def cityreader_stretch(lat1, lon1, lat2, lon2, cities=[]):
   # within will hold the cities that fall within the specified region
   within = []
-  
+
+  # make rectangle standardized with lower right and upper left coordinates.
+  lower_right_lat = min(lat1, lat2)
+  upper_left_lat = max(lat1, lat2)
+  lower_right_lon = max(lon1, lon2)
+  upper_left_lon = min(lon1, lon2)
```

Comment:
Really cool! This is the first solution I've seen so far that didn't try to swap the variables. I really like this approach, very intuitive.